x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -226,7 +226,9 @@ describe('Service Overview', () => {
         'suggestionsRequest'
       );
 
-      cy.get('[data-test-subj="environmentFilter"]').type('production');
+      cy.get('[data-test-subj="environmentFilter"] input').type('production', {
```

Comment:
Is this change needed?

Reply:
The APM Cypress tests failed 4 times in a row without it, which technically I can ask KibanaOps to skip the tests for us, but the failure looked solvable and reasonable so I figured I'd add it in. The test _is_ flaky, but I think this resolves that.

Reply:
I see, thanks for doing that!